### ISSUE TITLE
JDK8 JIT Server : testJITServer_1_FAILED with Segmentation error vmState=0x00051aff

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse/openj9#8788]
Failure link
------------

From an internal build `Test_openjdk8_j9_sanity.functional_x86-64_linux_jit_Nightly/11/`
```
22:45:06  openjdk version "1.8.0_252-internal"
22:45:06  OpenJDK Runtime Environment (build 1.8.0_252-internal-jenkins_2020_03_07_18_15-b00)
22:45:06  Eclipse OpenJ9 VM (build ibm_sdk-c65837788d, JRE 1.8.0 Linux amd64-64-Bit Compressed References 20200307_11 (JIT enabled, AOT enabled)
22:45:06  OpenJ9   - c65837788d
22:45:06  OMR      - 6c47a242c
22:45:06  JCL      - f80a8d6dd10 based on jdk8u252-b05)
```

Optional info
-------------

Failure output (captured from console output)
---------------------------------------------

```
Unhandled exception
Type=Segmentation error vmState=0x00051aff
J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
Handler1=00007FF3D752AE70 Handler2=00007FF3D6E1C9F0 InaccessibleAddress=0000000001A01F25
RDI=0000000002587E90 RSI=0000000001A01F00 RAX=0000000001A01F00 RBX=00007FF2A1CA1000
RCX=0000000000000000 RDX=000000000252FA30 R8=0000000000000008 R9=0101010101010101
R10=00007FF3D0450C38 R11=00007FF3D7EEBB7A R12=0000000000000004 R13=00007FF3A00181A0
R14=00007FF2A2258760 R15=0000000000000000
RIP=00007FF3CF6A4103 GS=0000 FS=0000 RSP=00007FF3CDAB14A0
EFlags=0000000000010246 CS=0033 RBP=00007FF2A2156640 ERR=0000000000000004
TRAPNO=000000000000000E OLDMASK=0000000000000000 CR2=0000000001A01F25
xmm0 000000003f800000 (f: 1065353216.000000, d: 5.263544e-315)
xmm1 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm2 ffffffffff000000 (f: 4278190080.000000, d: -nan)
xmm3 00ff000000000000 (f: 0.000000, d: 7.063274e-304)
xmm4 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm5 4330000000000000 (f: 0.000000, d: 4.503600e+15)
xmm6 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm7 4018000000000000 (f: 0.000000, d: 6.000000e+00)
xmm8 ffffffff00000000 (f: 0.000000, d: -nan)
xmm9 0000000000000000 (f: 0.000000, d: 0.000000e+00)
xmm10 00ff000000ff0000 (f: 16711680.000000, d: 7.063274e-304)
xmm11 0000000049d70a38 (f: 1238829568.000000, d: 6.120632e-315)
xmm12 000000004689a022 (f: 1183424512.000000, d: 5.846894e-315)
xmm13 0000000047ac082f (f: 1202456576.000000, d: 5.940925e-315)
xmm14 0000000048650dc0 (f: 1214582272.000000, d: 6.000833e-315)
xmm15 0000000046b73e38 (f: 1186414080.000000, d: 5.861665e-315)
Module=/home/jenkins/workspace/Test_openjdk8_j9_sanity.functional_x86-64_linux_jit_Nightly/openjdkbinary/j2sdk-image/j2sdk-image/jre/lib/amd64/compressedrefs/libj9jit29.so
Module_base_address=00007FF3CF4CD000

Unhandled exception in signal handler. Protected function: writeCrashDataToConsole (0x0)

----------- Stack Backtrace -----------
(0x00007FF3CF6A4103 [libj9jit29.so+0x1d7103])
(0x00007FF3CF69C4EE [libj9jit29.so+0x1cf4ee])
(0x00007FF3CF8A0527 [libj9jit29.so+0x3d3527])
(0x00007FF3CF8A11D7 [libj9jit29.so+0x3d41d7])
(0x00007FF3CF8C0807 [libj9jit29.so+0x3f3807])
(0x00007FF3CF8C156A [libj9jit29.so+0x3f456a])
(0x00007FF3CFC03577 [libj9jit29.so+0x736577])
(0x00007FF3CFC03A99 [libj9jit29.so+0x736a99])
(0x00007FF3CFC03A99 [libj9jit29.so+0x736a99])
(0x00007FF3CFC03A99 [libj9jit29.so+0x736a99])
(0x00007FF3CFC03A99 [libj9jit29.so+0x736a99])
(0x00007FF3CFC04EDB [libj9jit29.so+0x737edb])
(0x00007FF3CF9F4BFD [libj9jit29.so+0x527bfd])
(0x00007FF3CF653D67 [libj9jit29.so+0x186d67])
(0x00007FF3CF654CE1 [libj9jit29.so+0x187ce1])
(0x00007FF3D6E1D753 [libj9prt29.so+0x1b753])
(0x00007FF3CF6569B5 [libj9jit29.so+0x1899b5])
(0x00007FF3CF7E0422 [libj9jit29.so+0x313422])
(0x00007FF3CF652A83 [libj9jit29.so+0x185a83])
(0x00007FF3CF652EC2 [libj9jit29.so+0x185ec2])
(0x00007FF3CF652F6A [libj9jit29.so+0x185f6a])
(0x00007FF3D6E1D753 [libj9prt29.so+0x1b753])
(0x00007FF3CF6533BD [libj9jit29.so+0x1863bd])
(0x00007FF3D728B326 [libj9thr29.so+0xe326])
(0x00007FF3D8203AA1 [libpthread.so.0+0x7aa1])
clone+0x6d (0x00007FF3D7F50C4D [libc.so.6+0xe8c4d])
---------------------------------------
```

To rebuild the failed tests in <Jenkins URL>=https://ci.adoptopenjdk.net/job/Grinder, use the following links:
01:07:22  https://ci.adoptopenjdk.net/job/Grinder/parambuild/?JDK_VERSION=8&JDK_IMPL=openj9&BUILD_LIST=functional&PLATFORM=x86-64_linux&TARGET=testJITServer_1

**Comment by pshipton** [comment]
@mpirvu

**Comment by EmanElsaban** [comment]
I reran the test in grinder.It's failing in build stage: https://ci.adoptopenjdk.net/job/Grinder/4526/.
output:

**Comment by EmanElsaban** [comment]
Ran the test "testJITServer_1" 100 times on Grinder. It passes all tap tests: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/11834/

Output from console:
Java version:
```
07:02:10  Running make 4.1
07:02:10  set TEST_ROOT to /home/jenkins/workspace/Grinder/openjdk-tests/TKG/..
07:02:10  set JDK_VERSION to 8
07:02:10  set JDK_IMPL to openj9
07:02:10  set JVM_VERSION to openjdk8-openj9
07:02:10  set JCL_VERSION to latest
07:02:10  set JAVA_HOME to /home/jenkins/workspace/Grinder/openjdkbinary/j2sdk-image
07:02:10  set SPEC to linux_x86-64_cmprssptrs
07:02:10  Running testJITServer_1 ...
07:02:10  There are 1 test targets in testJITServer_1
07:02:10  "/home/jenkins/workspace/Grinder/openjdkbinary/j2sdk-image/bin/java" -version
07:02:10  openjdk version "1.8.0_272-internal"
07:02:10  OpenJDK Runtime Environment (build 1.8.0_272-internal-jenkins_2020_11_03_11_10-b00)
07:02:10  Eclipse OpenJ9 VM (build master-6c0fa8e436a, JRE 1.8.0 Linux amd64-64-Bit Compressed References 20201103_466 (JIT enabled, AOT enabled)
07:02:10  OpenJ9   - 6c0fa8e436a
07:02:10  OMR      - 4ede03ca616
07:02:10  JCL      - 8425b0a2dcf based on jdk8u272-b10)
```
FYI: @mpirvu

**Comment by dmitry-ten** [comment]
Probably fixed by #10408, although the VM state is different

**Comment by mpirvu** [comment]
Closing this issue based on the two comments above
